youtube_base/youtubers/admin.py

```diff
@@ -33,6 +33,19 @@ class CategoryAdmin(admin.ModelAdmin):
                     'description')
 
 
+class CommentAdmin(admin.ModelAdmin):
+    """The CommentAdmin class defines the admin interface for the Comment model."""
+    list_display = ('name',
```

Comment:
Why not to inline several tuple items as a single line?

Reply:
I just want to that the file has the same style.